app/javascript/mastodon/features/compose/components/compose_form.js

```diff
@@ -21,7 +21,7 @@ import ImmutablePureComponent from 'react-immutable-pure-component';
 import { length } from 'stringz';
 import { countableText } from '../util/counter';
 import Icon from 'mastodon/components/icon';
-import { maxChars } from '../../../initial_state';
+import { maxChars } from 'flavours/glitch/initial_state';
```

Comment:
We shouldn't mix flavours. I think the proper fix would rather be to add `maxChars` back to `app/javascript/mastodon/initial_state.js`

Reply:
I can't find it either :|
but I believe it was in a merge commit

Only `maxChars` was used, so only `maxChars` matters.